test_util/get_ci_user.py

```diff
@@ -0,0 +1,26 @@
+from test_util.auth_helper import get_cluster_auth_token
+from test_util.cluster_api import ClusterApi
+from test_util.helpers import DcosUser
+
+
+def get_cluster_api(dcos_url, masters, public_masters, slaves, public_slaves, dns_search_set, provider):
```

Comment:
This is a thin wrapper whose only purpose is inject the ci auth user. That is very terse logic, why do we need this helper?